### ISSUE TITLE
does not communicate any more

### ISSUE THREAD

**Issue by GoogleCodeExporter** [CarlitoBriganteNT/android-notifier#136]
```
What steps will reproduce the problem?
1. Install on phone, install on desktop
2. Excluded from norton!
3. Try test notification, anything


What is the expected output? What do you see instead?
It should produce something! It worked only once, yesterday.. it doesn't 
anymore. 
Oh yeah windows xp 32bit.

Please paste the application's log here (it should now be on your
clipboard):
Added as an attachment cause it was too long.

What version of Android are you using?
2.2 CM6 stable.. cyanogen's latest mod for N1


```

Original issue reported on code.google.com by `orestak...@gmail.com` on 13 Sep 2010 at 10:11
- Merged into: #135

**Comment by GoogleCodeExporter** [comment]
Original comment by `rdamazio@gmail.com` on 13 Sep 2010 at 5:13
- Changed state: **Duplicate**
